test/operators.jl

```diff
@@ -318,6 +319,14 @@
         @test @inferred(ndims(g)) == 2
     end
 
+    g10 = complete_graph(10)
+    @testset "Induced bipartite Subgraphs: $g" for g in testgraphs(g10)
+        sg = @inferred(induced_bipartite_subgraph(g, [2,3],[4,5]))
+        @test nv(sg) == 4
+        @test ne(sg) == 4
+        @test is_bipartite(sg)
+    end
```

Comment:
While this is good, I think we need a few more test cases, e.g.
* For empty graphs
* For graphs with self-loops
* For graphs that are not complete graphs
* For directed graphs
* For cases, where an exception is being thrown
* For `U` of different eltype than `Int`

It might also be good to test more  than just some properties, namely if the subgraph is actually the one that we wanted.

Reply:
I agree that more tests would be nice